src/integrationtest/python/should_write_setup_file_tests.py

```diff
@@ -47,6 +52,14 @@ def init (project):
 def spam ():
     pass
 """)
+        self.create_directory("src/main/python/project/descriptors/a")
+        self.create_directory("src/main/python/project/calendars/acc")
+
+        self.write_file("src/main/python/project/__init__.py", "")
```

Comment:
I always like condens writing, but I don't know your opinion about it.
My proposal would be to add a call to self.create_directory within this write_file (and write_file_binary) methods to ensure the directory structure is there.
WDYT?

Reply:
Looks fine to me :-). Sadly lot's of unrelated changes that polluted the review screen a bit (quote changes mainly)

Reply:
When I said "please review", I meant as a matter of "here's the example of directory structure and `__init__.py` files that produce the behavior you desire, which is further codified with a full integration test" :laughing:

Reply:
;-) This is how it is setup at our place yes. So nothing wrong with that there...

Reply:
Was it though? You had the `__init__.py` in the directory structures? Notice what you posted in #672:

![image](https://user-images.githubusercontent.com/5355666/77933692-16f73680-727d-11ea-97e5-43183368c426.png)

The `__init__.py` are not mentioned :smile:

Reply:
After you asked for verification (same ticket), I investigated the source at work and concluded:
![image](https://user-images.githubusercontent.com/199434/77949045-9d843600-72c6-11ea-889c-6a7e4ca92fdf.png)

So, the init's where actually there, but my memory didn't work very well at the former comment... (not unheard of :p)